frontend/src/components/ProductCreator/ProductCreator.js

```diff
@@ -49,13 +50,20 @@ const ProductCreator = ({ categoryId }) => {
   const [isProductLoading, setIsProductLoading] = useState(false);
   const [productLinkInputText, setProductLinkInputText] = useState('');
   const [isError, setIsError] = useState(false);
+  const [scrapeProduct] = useScrapeProduct();
 
   function onProductTextChange(event) {
     setProductLinkInputText(event.target.value);
     if (event.target.value === '') setIsError(false);
     setIsProductLoading(true);
     const linkUrl = event.target.value;
     if (isUrl(linkUrl)) {
+      if (linkUrl.includes('amazon')) {
```

Comment:
🔕 might want better determination of amazon links

Reply:
I can put this after we scrape the product to postgre. This way we'll know for sure if this is an amazon product.

Reply:
done